src/components/logger/logger.js

```diff
@@ -0,0 +1,88 @@
+import 'typeface-montserrat';
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import image from '../../assets/settings.png';
+import { defaultFontFamily } from '../constants';
+import Content from './content';
+
+const Container = styled.div`
```

Comment:
I think it's better to put styling in a separate file

Reply:
I appreciate your thoughts, but I do not agree)

First of all, there are not styles - it's a styled components - it's a code.

I prefer to split code if it can be:
- [ ] reusable.
- [ ] easy abstracted.
- [ ] consistent.

Long story short: 
```
if I want to cook an apple pie, I go to the shop and by ready dough and apples.
I don't want to buy sugar, yeast, vinegar, salt, 
canella etc and cook the dough at home.

If I will follow the approach that you propose me, 
I should split this apple pie into the components as much as possible and it can be continued to the splitting on the level of molecules. 

Somebody like likes molecular food, but not me
```

Just stop splitting something if you won't get more profit from it than spent efforts.

Reply:
It is more readable to me when there is only React component in one file and all the helpers that are used for styling purposes in a separate one, especially when there is quite much styling.
Your example is much too dramatic.
But it's not a big deal for me, I will not argue.